### PR TITLE
Mobile Responsive pass

### PR DESCRIPTION
**Pull request by kevinahuber**

### CODE REVIEW

File: src/screens/poa-form/styles.scss [hunk 1]
```diff
@@ -40,4 +40,21 @@
...
+    .button {
+      width: 100%;
+      max-width: 100%;
+      margin-bottom: 15px;
+    }
+  }
+}
+
+.stepper {
+  @media (max-width: 430px) {
```
Comment (kevinahuber) [inline src/screens/poa-form/styles.scss]: Hid the stepper once it starts to get stuffed.

**Comment by kevinahuber** [comment]
fixes #32

File: src/screens/poa-form/index.js [hunk 2]
```diff
@@ -420,7 +420,7 @@ class PoAForm extends React.Component {
                 {
                   title: t('guardianInformation'),
                   onClick: () => {
-                    this.setState(state => ({step: 1}))
+                    this.setState(staxwte => ({step: 1}))
```
Comment (mshenfield) [inline src/screens/poa-form/index.js]: Fix

File: src/screens/poa-form/styles.scss [hunk 3]
```diff
@@ -47,9 +47,27 @@
...
+.stepper {
+  div div div div {
+    // HACK: Since the slider sets all styles inline.
+    a,
+    span {
+      line-height: 32px !important;
+    }
+  }
+
+  @media (max-width: 430px) {
```
Comment (mshenfield) [inline src/screens/poa-form/styles.scss]: April requests that the "Back" button be hidden when the stepper _is_ visible (on larger screens).  I can file a separate issue for that though.

File: src/screens/poa-form/styles.scss [hunk 4]
```diff
@@ -47,9 +47,27 @@
...
 }
 
-.stepper div div div div { // HACK: Since the slider sets all styles inline.
-  a,
-  span {
-    line-height: 32px !important;
+.grommetux-box--direction-row.grommetux-box--responsive.button-box {
+  @media (max-width: 719px) {
+    flex-direction: column-reverse;
+    .button {
```
Comment (mshenfield) [inline src/screens/poa-form/styles.scss]: I think you could do this less hackily by wrapping the buttons in a [Grommet  Box](http://grommet.io/docs/box/).  Can you try making that work?

**Comment by mshenfield** [comment]
Looks good - had a few comments on better utilizing Grommet.

Have to run, but two other issues I noticed on mobile

- [ ] The "Number of Children" number isn't visible on small screens. 
- [ ] The form should reset the screen to the top of the page when clicking "Next".  Because some of the fields are identical and it doesn't reset, it's sometimes hard to tell when we made forward progress.

**Comment by CoreyAR** [comment]
Do you think we should add labels so that you know what form you are using?

File: src/screens/poa-form/styles.scss (hunk 4 above)
Comment (kevinahuber) [reply src/screens/poa-form/styles.scss]: It is wrapped in a grommet box- but in classic opinionated-but-not-opinionated-enough fashion, it doesn't really go the right route. imho, next should be at the top on mobile, which would warrant a `column-reverse` and full-width buttons. I don't think its _horrible_ without the hack, and am happy to remove if we want to keep it clean.

**Comment by kevinahuber** [comment]
@CoreyAR can you elaborate?

**Comment by mshenfield** [comment]
@CoreyAR Going to land - can you elaborate on the labels comment?  Will file an issue for the hidden "Number of Children" and can add one for the labels as well.

**Pull request merged**